### PR TITLE
docs: Added commands to README.md

### PR DESCRIPTION
**Pull request by Jbyford89**
Updated README.md with the roll, choose, and docs commands and a short description.

Also, added Owner Commands section with the roles, and add_confirm commands as well as a description.

Added an Owner Context Menu Commands section with the giveaway command and description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,12 +15,30 @@ Bot for the Dev Pro Tips YouTube Discussion and Support Server
 
 * `!help` - displays more info about commands
 
+* `!roll` - roles a given amount of dice format `_d_`
```
Comment (DenverCoder1) [inline README.md:18]:
```suggestion
* `!roll` - roles a given amount of dice in the format `_d_`
```

File: README.md [hunk 2]
```diff
@@ -15,12 +15,30 @@ Bot for the Dev Pro Tips YouTube Discussion and Support Server
...
+
+* `!choose` - chooses an item randomly from a list
+
+* `!docs` - fetch various python library documentation use `!help docs` for more info
```
Comment (DenverCoder1) [inline README.md:22]:
```suggestion
* `!docs` - fetch documentation for various Python libraries. Use `!help docs` for more info.
```

File: README.md [hunk 3]
```diff
@@ -15,12 +15,30 @@ Bot for the Dev Pro Tips YouTube Discussion and Support Server
...
 * **Welcome** - Sends a welcome message to users and gives them the Unassigned role
 
 * **Button Roles** - Assigns roles when users click the button components
 
 * **YouTube Feed** - posts new videos from my YouTube channel in `#videos`
 
+
+### Owner Commands
+
+* `!roles` - start a roles view, add or remove a role
```
Comment (DenverCoder1) [inline README.md:33]:
```suggestion
* `!roles` - start a role view for users to add and remove their roles
```

File: README.md [hunk 4]
```diff
@@ -15,12 +15,30 @@ Bot for the Dev Pro Tips YouTube Discussion and Support Server
...
+
+* `!add_confirm` - start a confirm view, confirm role was added
```
Comment (DenverCoder1) [inline README.md:35]:
```suggestion
* `!add_confirm` - add a confirm button to a given message
```

File: README.md [hunk 5]
```diff
@@ -15,12 +15,30 @@ Bot for the Dev Pro Tips YouTube Discussion and Support Server
...
+
+### Owner Context Menu Commands
+
+* `giveaway` - start and create a giveaway
```
Comment (DenverCoder1) [inline README.md:39]:
```suggestion
* `giveaway` - start and select winners for a giveaway
```

**Review by DenverCoder1 — CHANGES_REQUESTED** [review]
Looks good, thanks!

I have a few suggestions on the wording.

Also `!update_rules` can be moved to the Owner Commands section

File: README.md [hunk 6]
```diff
@@ -9,18 +9,36 @@ Bot for the Dev Pro Tips YouTube Discussion and Support Server
 
 * `!markdown` - displays Discord markdown tips
 
-* `!update_rules` - mod command for updating the embed containing the rules
-
 * `!ping` - check for a response from the bot
 
 * `!help` - displays more info about commands
 
+* `!roll` - roles a given amount of dice in the format `_d_`
```
Comment (DenverCoder1) [inline README.md:16]:
```suggestion
* `!roll` - rolls a given amount of dice in the format `_d_`
```

**Comment by Jbyford89** [comment]
If there is anything else I can do like this, I am more than happy to keep helping. Also helps improve my skill set as well.

Thank you for letting me contribute to your project.

**Pull request merged**